setup.py

```diff
@@ -168,7 +168,7 @@ def _main():
             "build_ext": BuildExtension.with_options(no_python_abi_suffix=True),
             "clean": clean,
         },
-        install_requires=[pytorch_package_dep, "numpy", "packaging"],
+        install_requires=[pytorch_package_dep, "numpy", "packaging", "PyOpenGL==3.1.0"],
```

Comment:
I am not happy with this but it seems version `3.1.6` still has issues that couldn't be resolved. More on: https://github.com/NixOS/nixpkgs/issues/76822.